db/migrate/20200425152028_add_img_url_to_article.rb

```diff
@@ -0,0 +1,5 @@
+class AddImgUrlToArticle < ActiveRecord::Migration[6.0]
```

Comment:
I cant see that you use this anywhere and it is also out of scope for this feature, please remove it.